index.css

```diff
@@ -51,6 +57,99 @@ body {
   }
 }
 
+footer > div > ul > li {
+  margin-left: 15px;
+}
+
+.about_lang > ul > li {
+  list-style-type: none;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  color: #fff;
+  margin-top: 20px;
+}
+
+.headline > div > ul > li {
+  display: inline;
+}
+
+footer > div > ul {
+  display: flex;
+  list-style-type: none;
+  padding-left: 0;
+  justify-content: center;
+  margin-top: 25px;
+  margin-bottom: 25px;
+}
+
+.works_det > div > ul {
+  margin-top: -10px;
+  margin-left: -25px;
+}
+
+#sideMenu {
```

Comment:
- [ ] The hamburger menu button is displayed on the desktop view, please fix it.

![mmm](https://user-images.githubusercontent.com/61562626/140939579-3820d2b9-945b-4429-b64a-8e20cc85bbcb.gif)

Reply:
Hello @dasileker I fixed that after you marked my request review invalid.
I think you didnot update your local file or forgot to clone it again. Here is the screen shot
![hamburgscreenshot](https://user-images.githubusercontent.com/42030948/140942375-c6c7022a-e127-4702-86f8-24f85d13c20e.png)

Reply:
we don't check the codes if the linters are passing so after fixing linters we are supposed to download the repo and make the review, if I didn't clone how I'm supposed to see the issue, but anyway.